frontend/packages/kubevirt-plugin/src/constants/vm/provision-source.ts

```diff
@@ -103,9 +103,13 @@ export class ProvisionSource extends ValueEnum<string> {
           return {
             error: `Datavolume ${volumeWrapper.getDataVolumeName()} does not have a supported source (${type}).`,
           };
+        case undefined:
```

Comment:
we could have a special check like this here, but it should go right after the volume lookup.